src/EventuallyConsistentTestTrait.php

```diff
@@ -27,13 +27,16 @@
  */
 trait EventuallyConsistentTestTrait
 {
-    /* @var int The number of retries for eventually consistent tests */
-    protected static $eventuallyConsistentRetryCount = 3;
-
-    private function runEventuallyConsistentTest(
-        callable $func,
-        $retries = self::$eventuallyConsistentRetryCount
-    ) {
+    private function runEventuallyConsistentTest(callable $func, $retries = 0)
+    {
+        if ($retries <= 0) {
+            // It doesn't make sense, we'll use the default value. You can
+            // also define the static property $eventuallyConsistentRetryCount in
+            // your concrete class then we'll use that value.
+            $retries = isset(static::$eventuallyConsistentRetryCount)
```

Comment:
Also, would prefer the function declaration to be `$retries = null` and the conditional to be`if (is_null($retries))`, as it's conceivable that someone wants to conditionally allow no retries.